packages/paste-website/src/pages/components/button/index.mdx

```diff
@@ -389,6 +389,49 @@ Use icon-only Buttons sparingly. If there is any text in a button, do not use th
 </Stack>`}
 </LivePreview>
 
+#### Circle Buttons
+
+You can create icon Buttons from variants that have background colors and borders,
+like primary and secondary, by using that variant and changing the size of the Button
+component to `size="circle"` or `size="circle_small"`.
+
+Circle buttons are similar to icon Buttons where they can only have an icon in them.
+To change the size of the Button, change the size of the Icon itself.
```

Comment:
suggestion: it'd be useful to link to the icons doc page here that talks about icon sizing https://paste.twilio.design/components/icons/usage-guidelines#resizing-icons